chef_master/source/release_notes_server.rst

```diff
@@ -11,13 +11,90 @@ Chef is a systems and cloud infrastructure automation framework that makes it ea
 
 .. end_tag
 
+What's New in 12.15
+=====================================================
+The following items are new for Chef server 12.15:
+
+* **Supports SUSE Linux Enterprise on x86_64**
+* **Add required_recipe endpoint**
+* **ACLs and groups can refer to global groups**
+* **User customization of field mapping**
+
+Supports SUSE Linux Enterprise Server on x86_64
+=====================================================
+Support for a new platform was added: SUSE Linux Enterprise Server 11 & 12 on x86_64.
+
+Add required_recipe endpoint
+=====================================================
+Added the ability to serve a required recipe file to chef-clients. 
+
+The setting ``required_recipe["enable"]`` in chef-server.rb enables the required recipe feature.
+
+The setting ``required_recipe["path"]`` in chef-server.rb specifies the recipe file to serve.
+
+The ``/organizations/<orgname>/required_recipe`` endpoint returns 404 for all organizations by default. It returns 401 when the request is not made by a client from the requested org and the feature is enabled.
+
+The ``/organizations/<orgname>/required_recipe`` endpoint returns the required recipe and 200 only when the endpoint is enabled and requested by an authorized client.
+
+See `Chef RFC 89 <https://github.com/chef/chef-rfc/blob/master/rfc089-server-enforced-recipe.md>`_ for a complete description on the ``required_recipe`` endpoint.
+
+ACLs and groups can refer to global groups
+=====================================================
+The server-admins group is useful, but it breaks roundtripping when it appears in an organizations ACLs and groups. This makes it difficult when using the API for backups.
+
+A new syntax was added for referring to global objects from org local context: ``ORGNAME::name and for global objects ::name``. This can, and is omitted wherever the context is clear. So, if the server-admins appears in an organizations ACL, you will see the name ``::server-admins``.
```

Comment:
I read these two sentences several times and it looks like some context is missing. Can you add some additional wording here? For example "This can, and is omitted wherever the context is clear" needs to be reworded.